### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nick-fedesna**
- include link to downloadable binary
- fix link to Android plugin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +36,7 @@ TrueColors and the included CyndiLauper library are both released under the MIT
 
 ## Related Projects
 - [DevsJustWantToHaveFun](https://github.com/vokal/DevsJustWantToHaveFun-macOS): A Mac tool to generate iOS code from the design elements in a `.truecolors` file.
-- [android-res-gen](https://github.com/vokal/android-res-gen): Automatic resource exporter plugin for android projects, including generating styles from a `.truecolors` file.
+- [TrueColors-Android](https://github.com/vokal/TrueColors-Android): Gradle plugin to generate resources for android projects, including  styles, colors and dimensions from a `.truecolors` file.
```
Comment (vokal-isaac) [inline README.md]: Comma after `colors`.

File: README.md [hunk 2]
```diff
@@ -7,6 +7,10 @@
 
 A Mac app for designers to specify colors, fonts, and metrics in a standardized format for developers to consume.
 
+Download latest build here:
+[https://rink.hockeyapp.net/apps/e2a44471302146ef8a121891f117f358](https://rink.hockeyapp.net/apps/e2a44471302146ef8a121891f117f358)  
+(The app will check for updates its self after initial install.)
```
Comment (vokal-isaac) [inline README.md]: s/`its self`/`itself`/

File: README.md [hunk 3]
```diff
@@ -7,6 +7,10 @@
 
 A Mac app for designers to specify colors, fonts, and metrics in a standardized format for developers to consume.
 
+Download latest build here:
+[https://rink.hockeyapp.net/apps/e2a44471302146ef8a121891f117f358](https://rink.hockeyapp.net/apps/e2a44471302146ef8a121891f117f358)
```
Comment (vokal-isaac) [inline README.md]: How about
`[Download the latest build from HockeyApp](https://rink.hockeyapp.net/apps/e2a44471302146ef8a121891f117f358)`?

**Review by vokal-isaac — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nick-fedesna) [reply README.md]: I am against the oxford comma, it's not how I was raised.

Comment (vokal-isaac) [reply README.md]: You can be against it all you want, but I stick to the Chicago Manual of Style, which says to use the serial comma. 😀

**Review by vokal-isaac — APPROVED** [review]

**Pull request merged**